CodeChallengeTests/Base/CodeChallengeTestCase.swift

```diff
@@ -38,18 +38,22 @@ class CodeChallengeTestCase: XCTestCase {
         formatter.numberStyle = .decimal
         formatter.minimumFractionDigits = 5
         formatter.maximumFractionDigits = 5
+
         for (i, accResult) in results.enumerated() {
             var name = accResult.name
             // try to make the results line up a bit by inserting the correct amount of tabs between the name and the avg based on the maximum name length vs this name's length
             if name.characters.count < maxNameLength {
                 let diff = maxNameLength - name.characters.count
                 let tabWidth = 5
-                let tabsToInsert = Int(ceil(Double(diff)/Double(tabWidth)))
+                var tabsToInsert = Int(ceil(Double(diff)/Double(tabWidth)))
+                if diff % tabWidth == 0 {
```

Comment:
really I think this hack should be done using spaces, not tabs anyway, not sure why I did that that way in the first place. But I didn't wanna refactor that bad in this change